Python/AppTokenPythonExample.py

```diff
@@ -102,12 +112,16 @@ def sign_request(request: requests.Request) -> requests.PreparedRequest:
     prepared_request.headers['X-App-Access-Sig'] = signature.hexdigest()
     return prepared_request
 
- # Such actions are presented below:
- # 1) Creating an applicant
- # 2) Adding a document to the applicant
- # 3) Getting applicant status
- # 4) Getting access token
-createApplicant()
-addDocument()
-getApplicantStatus()
-getAccessToken()
+
+# Such actions are presented below:
+# 1) Creating an applicant
+# 2) Adding a document to the applicant
+# 3) Getting applicant status
+# 4) Getting access token
+external_user_id = str(uuid.uuid4())
+level_name = 'basic-kyc-level'
+
+create_applicant(external_user_id, level_name)

```

Comment:
it would be better to see smth like this:
```
def main():
    external_user_id = str(uuid.uuid4())
    level_name = 'basic-kyc-level'
    applicant_id = create_applicant(external_user_id, level_name)
    print(applicant_id)
    image_id = add_document()
    print(image_id)
    status = get_applicant_status(applicant_id)
    print(status)
    token = get_access_token(external_user_id, level_name)
    print(token)

if __name__ == '__main__':
    exit(main())

Reply:
and functions should use logging instead of print inside.

Reply:
done.